src/packs/melee/Stun baton.yml

```diff
@@ -8,9 +8,6 @@ data:
     suppress: false
     type: none
     value: 0
-  bundle:
-    amount: 1
-    bundled: false
```

Comment:
Don't remove data unrelated to PR

Reply:
You helpfully did this in one of your commits.